src/Tests/Meditation.TestBootstrapManaged/Meditation.TestBootstrapManaged.csproj

```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
[Suggestion] Can we move all such configs to Directory.Build.props?

Reply:
It is in Directory.Build.props. These are explicit overrides that are utilized only by a couple of libraries (mainly tests)